cmd/cstor-pool-mgmt/app/command/commands_test.go

```diff
@@ -1,3 +1,17 @@
+// Copyright © 2017-2019 The OpenEBS Authors
```

Comment:
Can we just have 2017 here @kmova if possible, as i seen in other projects as well( kubernetes etc..), they mentioned only the year when the file has been created.

Reply:
Good point! we need to automate the validation of dates as well.